pkgs/tools/misc/bepasty/default.nix

```diff
@@ -18,10 +19,41 @@ let
     };
   };
 
-#We need to use buildPythonPackage here to get the PYTHONPATH build correctly.
-#This is needed for services.bepasty
-#https://github.com/NixOS/nixpkgs/pull/38300
-in with bepastyPython.pkgs; buildPythonPackage rec {
+  inherit (lib) licenses maintainers;
+
+  inherit (bepastyPython.pkgs)
```

Comment:
Hm no, quite clearly I'd put the `with bepastyPython.pkgs;` in front of the `buildInputs` (and such) lists. If you want to be fancy, you can do some `attrValues { inherit (...) ...; }`, but that amounts to the same thing in this particular case.